node/pkg/driver/node_utils.go

```diff
@@ -230,11 +238,37 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
-func (n NodeUtils) Exists(path string) bool {
+func (n NodeUtils) IsPathExists(path string) bool {
 	_, err := os.Stat(path)
 	if err != nil {
+		if !os.IsNotExist(err) {
+			return false
+			logger.Warningf("Check is file %s exists returned error %s", path, err.Error())
+		}
 		return false
 	}
 
 	return true
 }
+
+func (n NodeUtils) IsDirectory(path string) bool {
+	targetFile, err := os.Stat(path)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			logger.Warningf("Check is directory %s returned error %s", path, err.Error())
+		}
+		return false
+	}
+	return targetFile.Mode().IsDir()
+}
+
+// Deletes file or directory with all subdirectory and files
+func (n NodeUtils) RemoveFileOrDirectory(path string) error {
+	return os.RemoveAll(path)
+}
+
+// path: file/dir path
+// return: path to the file/dir if accessed from pod
```

Comment:
https://github.com/IBM/ibm-block-csi-driver/pull/117#discussion_r373863668
I don't understand/agree, comment. don't silently resolve without changing

Reply:
what did you mean here? "..and ..."

Reply:
I did not use the word "and". I just provided the entire comment line, that I think should be, instead of the current one

Reply:
Please specify what's wrong here. I don't read thoughts.

Reply:
There shouldn't be an `if` there, because it is not a question of whether the path will be accessed from within the pod or not, since it **always** returns the path.
`from pod` is not grammatically correct, because singular common nouns **always** require an article before them.
Hence, it is a path **to be** used from **the**(/a) pod.
You don't have to read thoughts, it is a short sentence, with a clear difference.
If you reject my version, you should be the one specifying why.

Reply:
I don't agree. Leave as is

Reply:
then ask for a second opinion